zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/jsProductionTasks.kt

```diff
@@ -49,7 +49,7 @@ internal fun JsIrBinary.asJsProductionTask(): JsProductionTask {
     override val targetName get() = target.name
     override val toolName = null
     override val mode get() = this@asJsProductionTask.mode
-    override val outputFile get() = linkTask.map { File(it.kotlinOptions.outputFile!!) }
+    override val outputFile get() = linkTask.map { it.outputFileProperty.get() }
```

Comment:
use `flatMap`

Reply:
I tried `flatMap` earlier, but it [fails the build](https://github.com/cashapp/zipline/pull/868/commits/07fcb0d10514b0d8fc98e23d42030073817f77bb).

Reply:
Oh, okay, strange 😅